lib/cdo/app_server_hooks.rb

```diff
@@ -34,6 +38,9 @@ def self.after_fork(host:)
       # intialized in config/initializers/statsig.rb
       require 'cdo/statsig'
       Cdo::StatsigInitializer.init
+
+      require 'cdo/app_server_metrics'
+      Cdo::AppServerMetrics.start_background_metrics_thread(host: host)
```

Comment:
Can we exclude all systems except the managed test server and production (`CDO.rack_env?(:production) || CDO.test_system?`)? I think we use puma on development and continuous integration builds, so that's a lot of Metrics (many of which share the same `host`).